connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -668,6 +670,12 @@ export async function githubRepoGarbageCollectActivity(
       repoId: repoId.toString(),
     },
   });
+
+  await deleteFolderNode({
+    dataSourceConfig,
+    folderId: repoId,
+    loggerArgs: logger.bindings(),
```

Comment:
2 points here:
1. what led you to pass logger.bindings here? not sure what this does
2. in general, this argument is now not necessary, you can remove it from the call and even delete it from `deleteFolderNode`

Reply:
1. I added it because it is used in upsertDataSource. Indeed currently not used in deleteFolderNode (although it is a parameter), but thought that if we start using it, at least we don't have to modify this call to pass it.  I'm ok with deleting it 👍